### PR TITLE
Update hash

### PR DESCRIPTION
**Pull request by hst12**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/167986)

### CODE REVIEW

**Review by hst12 — COMMENTED** [review]
Update hash again

File: manifests/h/hst/StarfieldTools/1.7.2/hst.StarfieldTools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+PackageIdentifier: hst.StarfieldTools
+PackageVersion: 1.7.2
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: msi
+NestedInstallerFiles:
+- RelativeFilePath: Starfield Tools Setup.msi
+ProductCode: '{86B0027A-266D-4E39-B428-2C251EBB44E0}'
+Installers:
```
Comment (stephengillie) [inline manifests/h/hst/StarfieldTools/1.7.2/hst.StarfieldTools.installer.yaml:12]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @hst12,

This package has a dependency on: `Microsoft.DotNet.DesktopRuntime.8`:
image
image

**Comment by stephengillie** [comment]
Verify:
image
It appears that `Starfiedl` is a dependency. Unfortunately, I don't see a package with `Starfield` currently. in our repo. I'm not sure how feasible it would be to get `Starfield` in the repo, so I'll approve the manifest without it. 
image

**Review by stephengillie — APPROVED** [review]

**Comment by hst12** [comment]
> Verify: image It appears that `Starfiedl` is a dependency. Unfortunately, I don't see a package with `Starfield` currently. in our repo. I'm not sure how feasible it would be to get `Starfield` in the repo, so I'll approve the manifest without it. image

Thanks. It installed and is working correctly on my side.

**Pull request merged**